### ISSUE TITLE
Support for CI environments is lacking

### ISSUE THREAD

**Issue by kibertoad** [jorgebucaran/colorette#61]
Current implementation doesn't reliably determine colour support in CI environments, resulting e. g. in CI for https://github.com/pinojs/pino-pretty/tree/feat/colorette breaking.

I think these lines from `supports-colour` should be included:

```
	if ('CI' in env) {
		if (['TRAVIS', 'CIRCLECI', 'APPVEYOR', 'GITLAB_CI', 'GITHUB_ACTIONS', 'BUILDKITE', 'DRONE'].some(sign => sign in env) || env.CI_NAME === 'codeship') {
			return 1;
		}

		return min;
	}
```

**Comment by kibertoad** [comment]
@jorgebucaran Would you accept a PR for this?

**Comment by jorgebucaran** [comment]
Of course. Do you think we can simplify this expression to check for only the main services (GitHub, CircleCI)?

**Comment by kibertoad** [comment]
@jorgebucaran Sure, although I would also include GitLab there, lots of major players are using it.

**Comment by kibertoad** [comment]
@jorgebucaran Any chance you could publish a new version?

**Comment by jorgebucaran** [comment]
Yes, sorry, will do! 🙇‍♂️

**Comment by jorgebucaran** [comment]
Published as @1.3.0. I didn't include #60 because it doesn't really block you.
